digitalstrategie/templates/base.html

```diff
@@ -11,7 +11,22 @@
         <link rel="icon" type="image/png" sizes="32x32" href="{% static 'images/favicon-32x32.png' %}">
         <link rel="icon" type="image/png" sizes="96x96" href="{% static 'images/favicon-96x96.png' %}">
         <link rel="icon" type="image/png" sizes="16x16" href="{% static 'images/favicon-16x16.png' %}">
+
         <link rel="stylesheet" type="text/css" href="{% static 'digitalstrategie.css' %}" />
+
+        <!-- Berlin theme start -->
+            <link rel="stylesheet" type="text/css" href="//www.berlin.de/i9f/r1/bundle/berlin_marketing.css">
+            <link rel="stylesheet" type="text/css" href="//www.berlin.de/i9f/r1/bundle/swiper.min.css">
+            <link rel="stylesheet" type="text/css" href="//www.berlin.de/i9f/r1/bundle/shariff.min.css">
+            <link rel="stylesheet" type="text/css" href="//www.berlin.de/i9f/r1/bundle/fontawesome.all.min.css">
+
+            <script src="//www.berlin.de/i9f/r1/bundle/jquery.min.js"></script>
```

Comment:
we also load jquery through webpack, I wonder whether we should remove either this line or the webpack loading

Reply:
ahhh good catch, right I was actually going to remove it in the webpack as we weren't using it but as it's quite big I guess it's better to remove this one,webpack config makes it gloabally available and with all usual aliases so it shouldn't make a diffrence, i'll try and see, cheers!